aea/context/base.py

```diff
@@ -66,6 +68,16 @@ def public_keys(self) -> Dict[str, str]:
         """Get public keys."""
         return self._public_keys
 
+    @property
+    def addresses(self) -> Dict[str, str]:
+        """Get addresses."""
+        return self._addresses
+
+    @property
+    def address(self) -> str:
+        """Get the defualt address."""
+        return self._addresses['default']
```

Comment:
Why this returns only the default address?

Reply:
@marcofavorito for now we want to stay backwards compatible in all the skills to avoid a rewrite, hence `public_key` points to the default one. So to mirror that `address` does the same.

Reply:
In future, we can give the developer more control over what address exposed here.